telegram/bot.py

```diff
@@ -303,67 +287,68 @@ def _validate_token(token: str) -> str:
         return token
 
     @property  # type: ignore
-    @info
+    def bot(self) -> User:
+        """:class:`telegram.User`: Instance for the bot."""
+
+        if self._bot is None:
+            self.get_me()
```

Comment:
this is probably why everthing's  failing ;)

```suggestion
            self._bot = self.get_me()
```

Reply:
Oh i thought as `bot.get_me()` call sets attr anyway, so we can leave that...